server/src/EngineModel.ts

```diff
@@ -0,0 +1,82 @@
+import {
```

Comment:
Woops, I might have commented on the compiled version. Well, go back there to see what I wrote, I guess… xD